lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -185,29 +183,37 @@ class FullPageScreenshot extends FRGatherer {
     const session = context.driver.defaultSession;
     const executionContext = context.driver.executionContext;
     const settings = context.settings;
-
-    // In case some other program is controlling emulation, remember what the device looks
-    // like now and reset after gatherer is done.
-    let observedDeviceMetrics;
     const lighthouseControlsEmulation = !settings.screenEmulation.disabled;
+
+    /**
+     * In case some other program is controlling emulation, remember what the device looks like now and reset after gatherer is done.
+     * If we're gathering with mobile screenEmulation on (overlay scrollbars, etc), continue to use that for this screenshot.
+     * @type {{width: number, height: number, deviceScaleFactor: number, mobile: boolean}}
+     */
```

Comment:
nit: moving the comment into this jsdoc is a bit weird. I read normal `//` comments as narrative in the code flow, but jsdoc/block comments are tied to the variable/fn they are attached to (even the IDE will display this string when you hover over the variable).

these two lines of comments should probably go inside the `if (!lighthouseControlsEmulation)`, or just before it.